specs/protocol/tx_format.md

```diff
@@ -110,6 +110,7 @@ Transaction is invalid if:
 - Any output is of type `OutputType.ContractCreated`
 - `scriptLength > MAX_SCRIPT_LENGTH`
 - `scriptDataLength > MAX_SCRIPT_DATA_LENGTH`
+- `bytePrice != 0`
```

Comment:
I think we had talked about adding a consensus rule s.t. bytePrice == gasPrice * N. I don't think we should prevent fee-less txs if miners are willing to include them.

Reply:
We talked about both, but it is slightly better in my oppinion to have zeroed bytePrice as it can be joker field in future if needed. But it does not matter a lot both are consensus rule change.

Reply:
Ah I see, I misread this as requiring bytePrice to be non-zero instead of always setting it to zero.

Reply:
Yea that rule logic is kinda weird

Reply:
I'm okay with either here, though I'll defer to @Voxelot who says we should use [`bytePrice = gasPrice * GAS_PER_BYTE`](https://github.com/FuelLabs/fuel-specs/pull/281#issuecomment-1048331577).